### PR TITLE
#2047 backport

### PR DESCRIPTION
**Pull request by 3lnc**
## Steps

- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Add a ChangeLog entry describing what your PR does.
- [x] If it's a new feature or an important bug fix, add a What's New entry in `doc/whatsnew/<current release.rst>`.
- [x] Write a good description on what the PR does.

## Description
Backporting fix for snake-case false positives from https://github.com/PyCQA/pylint/issues/2047

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :hammer: Refactoring  |

### CODE REVIEW

**Comment by 3lnc** [comment]
I have no idea what's happening with `DeprecationWarning` test. It shouldn't be affected at all, but it's broken for py2/pypy in travis (and works for py3), while passes in appveyor :man_shrugging:

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -164,3 +164,41 @@ Order doesn't matter (not that much, at least ;)
 * Jason Owen: contributor
 
 * Mitchell Young: minor adjustment to docparams
+
+* Marianna Polatoglou: minor contribution for wildcard import check
```
Comment (PCManticore) [inline CONTRIBUTORS.txt]: Can we remove these `CONTRIBUTORS` entries? I think most of them appeared only in 2.X so I'm not sure it makes sense to backport them to 1.9.

File: ChangeLog [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 ------------------
 
+What's New in Pylint 1.9.5?
+=========================
+
+Release date: |TBA|
+
+   * Fix a false positive `invalid name` message when method or attribute name is longer then 30 characters.
+
+       Close #2047
```
Comment (PCManticore) [inline ChangeLog]: The indentation seems a bit off.

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (3lnc) [reply CONTRIBUTORS.txt]: Fixed

File: ChangeLog (hunk 2 above)
Comment (3lnc) [reply ChangeLog]: As cherry-picked ¯\_(ツ)_/¯

Fixed

**Pull request merged**